Failure on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.44342&attempt_id=95fd7c86-f1cb-4449-ad41-1960a578aff1)

Attempted: tinyproxy

<details><summary>Partial log (click to expand)</summary><p>

```
a2x: ERROR: "xsltproc"  --stringparam callout.graphics 0 --stringparam navig.graphics 0 --stringparam admon.textlabel 1 --stringparam admon.graphics 0  "/nix/store/s2xd83mkhjmck3jf491mq96ygnrg2q4x-asciidoc-8.6.9/etc/asciidoc/docbook-xsl/manpage.xsl" "/build/source/docs/man5/tinyproxy.conf.xml" returned non-zero exit status 5
make[3]: *** [Makefile:522: tinyproxy.conf.5] Error 1
make[3]: Leaving directory '/build/source/docs/man5'
make[2]: *** [Makefile:342: all-recursive] Error 1
make[2]: Leaving directory '/build/source/docs'
make[1]: *** [Makefile:392: all-recursive] Error 1
make[1]: Leaving directory '/build/source'
make: *** [Makefile:333: all] Error 2
builder for '/nix/store/s2c6w8090706pvy5k8y4w77gpws40lic-tinyproxy-1.8.4.drv' failed with exit code 2
error: build of '/nix/store/s2c6w8090706pvy5k8y4w77gpws40lic-tinyproxy-1.8.4.drv' failed
```
</p></details>

--- a/pkgs/tools/networking/tinyproxy/default.nix
+++ b/pkgs/tools/networking/tinyproxy/default.nix
@@ -1,4 +1,5 @@
-{ stdenv, fetchFromGitHub, automake, autoreconfHook, asciidoc, libxml2, libxslt }:
+{ stdenv, fetchFromGitHub, automake, autoreconfHook, asciidoc, libxml2,
+  libxslt, docbook_xsl }:
 
 stdenv.mkDerivation rec{
   name = "tinyproxy-${version}";
@@ -11,7 +12,7 @@ stdenv.mkDerivation rec{
     owner = "tinyproxy";
   };
 
-  nativeBuildInputs = [ autoreconfHook asciidoc libxml2 libxslt ];
+  nativeBuildInputs = [ autoreconfHook asciidoc libxml2 libxslt docbook_xsl ];
 
   # -z flag is not supported in darwin
   preAutoreconf = stdenv.lib.optionalString stdenv.isDarwin ''
@@ -24,11 +25,13 @@ stdenv.mkDerivation rec{
   postConfigure = ''
     substituteInPlace docs/man5/Makefile --replace \
           "-f manpage" \
-          "-f manpage \\
+          "--xsltproc-opts=--nonet \\
+           -f manpage \\
            -L"
     substituteInPlace docs/man8/Makefile --replace \
           "-f manpage" \
-          "-f manpage \\
+          "--xsltproc-opts=--nonet \\
+           -f manpage \\
            -L"
   '';
 

